eahub/profiles/models.py

```diff
@@ -254,6 +254,9 @@ class Profile(models.Model):
     )
     city_or_town = models.CharField(max_length=100, blank=True)
     country = models.CharField(max_length=100, blank=True)
+    linkedin_url = models.CharField(max_length=400, blank=True)
```

Comment:
I think we should have some more validation here, i.e., whether the input is of a valid URL format. Django appears to have an [built-in URL class](https://docs.djangoproject.com/en/2.2/ref/models/fields/#urlfield), maybe that would be a good choice here